src/py/_pyodide/_core_docs.py

```diff
@@ -200,7 +200,7 @@ def assign(self, rhs: Any, /):
 
     # Argument should be a buffer.
     # See https://github.com/python/typing/issues/593
-    def assign_to(self, to: Any, /):
+    def assign_to(self, to: Any, /) -> None:
```

Comment:
```suggestion
    def assign_to(self, to: Any, /) -> Any:
```

Oops, meant to make these all Any. Wasn't sure if they could be made more specific (or if some of them are None, etc). They are in docs so no implementation was handy.